src/pages/Folder/components/FolderCard/index.jsx

```diff
@@ -0,0 +1,96 @@
+import styles from "./styles.module.css";
+import { getTimeAgo, formatDate } from "util/time";
+import noImg from "assets/noImg.png";
+import starImg from "assets/star.png";
+import kebabImg from "assets/kebab.png";
+import { useState } from "react";
+import { ModalPortal, PopOver, DeleteModal, AddLinkModal } from "components";
+import { useModal } from "hooks/useModal";
+
+export function FolderCard({ link, folders }) {
+  const createdAt = "createdAt" in link ? "createdAt" : "created_at";
+  const imageSource = "imageSource" in link ? "imageSource" : "image_source";
```

Comment:
P2:
변수명만 보면 키값이 아닌 값인것 같아요. 이렇게 키로 사용하실거면 `createAtKey`,`imageSourceKey`같은 식으로 바꾸시거나 아니면 아래처럼 하시면 좋겠습니다.

```suggestion
  const createdAt = "createdAt" in link ? link.createdAt : link["created_at"];
  const imageSource = "imageSource" in link ? link.imageSource : link["image_source"];
```